tests/unittest/Settings/Settings_test.cpp

```diff
@@ -0,0 +1,261 @@
+/*
+Copyright 2022 The Foedag team
+
+GPL License
+
+Copyright (c) 2022 The Open-Source FPGA Foundation
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "../../src/Main/Settings.h"
+
+#include <QStringList>
+#include <fstream>
+
+#include "Main/Tasks.h"
+#include "Main/WidgetFactory.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using ::testing::ElementsAre;
+
+namespace FOEDAG {
+namespace {
+
+TEST(Settings, loadSettings_merges_json) {
```

Comment:
Please update the test case name as per gtest's recommendation.
https://google.github.io/googletest/primer.html#simple-tests